base_cron_exclusion/views/ir_cron_view.xml

```diff
@@ -1,13 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- Copyright 2017 Eficent Business and IT Consulting Services S.L.
```

Comment:
Why you are removing this copyrights, see template again: https://github.com/OCA/maintainer-tools/blob/master/template/module/views/model_name_view.xml

Reply:
Done.